recipe/meta.yaml

```diff
@@ -36,3 +38,4 @@ about:
 extra:
   recipe-maintainers:
     - Matt-BF
```

Comment:
Could you remove me as maintainer? I don't feel like I can contribute to the project, just thought it would be a good idea initially to add it to conda =)

Reply:
@Matt-BF  But I believe that your authorization will still be necessary to carry out future MRs since you were the one who added it to the conda